src/main/java/ch/njol/skript/classes/ClassInfo.java

```diff
@@ -277,6 +279,16 @@ public ClassInfo<T> documentationId(String id) {
 		return this;
 	}
 
+	/**
+	 * Sets the literal patterns of this {@link ClassInfo}
+	 * @param patterns The literal patterns
+	 * @return This {@link ClassInfo} object
+	 */
+	public ClassInfo<T> patterns(String[] patterns) {
+		this.patterns = patterns;
+		return this;
+	}
```

Comment:
I'm not sure i like having this be a method on ClassInfo itself. It implies that classinfos should have and will have patterns. ClassInfos may not have parsers or may have parsers that don't adhere to patterns. I think this should be a method on RegistryParser or a new Parser superclass like PatternedParser or something.